src/managers/NotionEventSyncManager.ts

```diff
@@ -0,0 +1,106 @@
+
+import { Service } from 'typedi';
+import schedule from 'node-schedule';
+import { BotClient } from '../types';
+import Logger from '../utils/Logger';
+import { syncHostFormToNotionCalendar } from '../event-notion';
+import { readFileSync } from 'fs';
+import { MessageEmbed, WebhookClient } from 'discord.js';
+import { GoogleSheetsSchemaMismatchError, NotionSchemaMismatchError } from '../types';
+
+/**
+ * NotionEventSyncManager manages the automatic import of new events on the Events
+ * Host Form to the private board Notion Events calendar every 30 minutes.
+ */
+@Service()
+export default class {
```

Comment:
We can consider renaming this manager to `NotionEventManager`, since this will handle more than just syncs (i.e. TAP and CSI deadline pings and key reminders, etc.) but that's a discussion for later.